src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/boss_razorgore.cpp

```diff
@@ -324,6 +334,12 @@ class spell_egg_event : public SpellScriptLoader
         {
             PrepareSpellScript(spell_egg_eventSpellScript);
 
+            void HandleScript(SpellEffIndex effIndex)
+            {
+                PreventHitDefaultEffect(effIndex);
+                if (Unit* target = GetHitUnit());
```

Comment:
?

Reply:
I'm still preparing the script, forgot not to commit this part